### DISCUSSION TITLE
pros and cons (this time: summary nodes break "usual" navigation behavior) and moving on

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#638] [General]
Hi all,
the more I dive into Freeplane, I see that:
(1) the program is much more powerful and mindmapping is much cooler than I had thought before - this is good!
(2) unfortunately on the other side, I get more and more frustrated with the "little" annoyances. Here is another one:

When I (the current selection) am on a summary node or a descendant of it - let's call it summary branch - and I press UP on the keyboard, then:
(1) ok, first the selection goes up to the next sibling node inside that summary branch, if there is any.
(2) when the currently selected node is already the visually "most upwards" sibling node in that branch on that level, the current selection does not go up to the next branch, as it does on a normal branch, nooooooo. It must go left in the same branch to the last node that is located right before the summary node.

Guys, this is not how is works well. Now I know why there seem to be not as many active users here in the forums as one could think by just looking at the powerful features alone. The problem is that the overall usability is just below a smoothly usable point. I can't deal with that on the long run. I guess I must switch to a paid mindmap software. But thanks to Freeplane and its developer(s) for helping me to discover the world of mind mapping more.

Regards, autoca

**Comment by euu2021** [comment]
>Now I know why there seem to be not as many active users here in the forums as one could think by just looking at the powerful features alone. 

I just checked the Mindjet Mindmanager (the only other "powerful" mindmapping software?) forum, and it's less active than this forum. Maybe, you have as a reference for forum activity a program of broader interest, like LibreOffice, or a trendy (heavy marketing$$$) one, like Notion. Mindmapping software is way more niche. 

If you find another mindmapping program that has a better overall usability, please tell me, so I can learn from it and, maybe, suggest changes that could improve Freeplane.

**Reply by autoca** [reply]
You are right. I apologize. After searching a bit myself, I must agree that Mindjet MindManager (owned by Corel) seems to be the only other similarly powerful mindmapping software on the market. Yet, it is currently quite expansive for a "general" office software: either >200 $/year as subscription or >400 $ as a one time payment for a perpetual license (without any further upgrades).

I did not know that. XMind and Mindomo are so well spoken of, but they lack so much functionality compared to Freeplane or MindManager - I am shocked.

Therefore, I come crawling back in the dust humbly repenting and throwing ashes on my head. I did not know that the market looks the way it does. I checked out your free product first. I hope you can forgive me and hope I did not say too many bad things - I tried to be fair and hope it was also perceived like that. Probably I was a bit unfair with the comment about the forum, you are right there too. It was a too emotional expression of my frustration. I apologize for that specifically.

**Regarding the actual topic of Usability and the behavior when using the arrow keys:**
As a matter of fact, I do perceive MM to be more usability friendly. I downloaded a demo/test version.
If you want to use it as a source for inspiration, I would certainly support that idea mentally.

Eg the originally mentioned navigation behavior with arrow keys when inside a "summary branch" makes much more sense to me.
But anyway, I will eventually try to look into scripting in order to adapt the navigation behavior to my needs, I guess, just to give some more feedback.

Still, I hope I am still allowed to post feature requests in the future. For now, I want to mention again that Freeplane is nice because of its powerful features in connection with the unbeatable price tag. Despite the lacking usability, thanks for developing it.

Regards, autoca

**Reply by dpolivaev** [reply]
Welcome back!
Freeplane is a non-profit community and passion-driven project.
I believe it is absolutely best-in-class. And it has many features not available anywhere else.
It is as good as we all together are passionate about it and can afford to make it.
Cheers, Dimitry

**Reply by euu2021** [reply]
I forgot to mention that I do agree with you that there is a lot of room for improvement of the overall usability. But, improvements are always happening, in an impressive speed (@dpolivaev is a beast!). 

Your suggestions are well reasoned and explained. Please, continue with them. [This is a guide](https://sourceforge.net/p/freeplane/discussion/758437/thread/0199d02db8/#5edf) explaining what makes a good suggestion.

>But anyway, I will eventually try to look into scripting in order to adapt the navigation behavior to my needs, I guess, just to give some more feedback.

Scripting is very useful to accommodate FreePlane to your use cases. Eventually, you will achieve a perfect match, and then things will run smooth as butter.

**Comment by dpolivaev** [comment]
> Guys, this is not how is works well. Now I know why there seem to be not as many active users here in the forums as one could think by just looking at the powerful features alone. The problem is that the overall usability is just below a smoothly usable point. I can't deal with that on the long run. I guess I must switch to a paid mindmap software. But thanks to Freeplane and its developer(s) for helping me to discover the world of mind mapping more.

Summary nodes are an original Freeplane feature. And usually, when I introduce a new feature I do not think about all possible use cases and behaviors. When they are discovered by the community users and if the use cases and expectations are clearly stated I come back and make improvements. We inspect and adapt and improve.

It gives the community the very special influence on the app development. It allows the addition of behaviors needed by people instead of speculating what the people might need.

So if you send mind map examples and express your expectations it is likely that they come true.

**Reply by autoca** [reply]
> So if you ... express your expectations it is likely that they come true.

Well, I would want the selection around summary nodes to behave exactly as it does around any other normal node. E.g. if a s-node is selected and I press the up arrow key, the selection should go to the next, either sibling or branch "above" it, and not to somewhere to the left. Stay consistent in behaviour.
